dev/tools/make-changelog.sh

```diff
@@ -35,7 +35,7 @@ if [ ! -z "$fixes_string" ]; then fixes_string="$(printf '\n  fixes %s,' "$fixes
 # the ` are regular strings, this is intended
 # use %s for the leading - to avoid looking like an option (not sure
 # if necessary but doesn't hurt)
-printf '%s **%s:**\n  Bla bla\n  (`#%s <https://github.com/coq/coq/pull/%s>`_,%s\n  by %s).' - "$type_full" "$PR" "$PR" "$fixes_string" "$(git config user.name)" > "$where"
+printf '%s **%s:**\n  Bla bla (don\'t end with period here)\n  (`#%s <https://github.com/coq/coq/pull/%s>`_,%s\n  by %s).' - "$type_full" "$PR" "$PR" "$fixes_string" "$(git config user.name)" > "$where"
```

Comment:
```suggestion
printf '%s **%s:**\n  Bla bla (don'\''t end with period here)\n  (`#%s <https://github.com/coq/coq/pull/%s>`_,%s\n  by %s).' - "$type_full" "$PR" "$PR" "$fixes_string" "$(git config user.name)" > "$where"
```

or perhaps we should just do
```suggestion
printf '%s **%s:**\n  Bla bla.\n  (`#%s <https://github.com/coq/coq/pull/%s>`_,%s\n  by %s)' - "$type_full" "$PR" "$PR" "$fixes_string" "$(git config user.name)" > "$where"
```

Reply:
But isn't the second one precisely what we don't want, or did I misunderstand?

Reply:
@Zimmi48 What do you think?

Reply:
I was the one who introduced the changelog entries and my impression was that it looked better without a period before the parenthesis. Now I'm not sure anymore, but if we decide that in the end we want a period there, then it's the period after the parenthesis that looks weird to me.

Reply:
cc @jfehrle

Reply:
We could do it either way, provided that we're consistent (though IMHO period after the parenthesis is more correct). 
If the text generated by the script is "Blah blah. (...)" with a period before `(` and no period after the `)`, I expect developers will generally be consistent about it.  It's probably harder to train developers not to have a period before the `(`.  This is not something I want spend a lot of time checking for and flagging in reviews, it's a small detail.

I suppose one could update `changes.rst` to make it consistent.

BTW, "bla" should be "blah" in English.